### PR TITLE
Admin tokens docs

### PR DESCRIPTION
**Pull request by djgrove**
## Description
Add documentation outlining the admin tokens feature.

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] I have manually confirmed that all new links work.

### CODE REVIEW

File: themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md
Comment (djgrove) [inline themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:118]: We typically don't include screenshots for the cloud console in the docs, so I've omitted. Could be helpful here for context but this also seems straightforward.

File: themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:113]: >Namelly, these can be used to manage Policy-as-Code, org membership, and export audit logs.

As written, this sounds like these are the _only_ things they can do, but I'm assuming that isn't the case, and that admin tokens allow you to do anything a regular org admin can do except create or delete org tokens. Is that right?

Comment (djgrove) [reply themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:113]: Good question. I couldn't find a way to word this that I liked yesterday. You have this correct. The intention was to provide a few examples of features this enables without requiring that the user navigate to another page to check what privileges administrators have.

Org tokens have the ability to generate team tokens. They cannot generate personal access tokens because this is not applicable (they aren't a real user account). They are prohibited from managing other organization tokens as a requirements choice/safeguard, though it's possible we might drop this restriction at some point.

Any suggestions to better communicate this?

Comment (cnunciato) [reply themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:113]: Ok cool, thanks for confirming. Would this maybe help?

```suggestion
Admin organization access tokens (or admin tokens) are organization tokens with elevated, administrator-level privileges. Admin tokens allow automated processes to perform any operation supported for organization administrators except for the creation or deletion of other organization tokens. 
```

This way, you can leave it to the docs to explain what's allowed and what isn't for administrators, and if y'all decide to remove the org-token restriction at some point, you can do that pretty easily. Feel free to accept or adjust -- just offering as something to consider if you like.

Comment (djgrove) [reply themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:113]: I like this, sounds good!

File: themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md
Comment (meagancojocar) [inline themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:117]:
```suggestion
Exercise caution and limit the use of admin organization access tokens to scenarios where they are absolutely necessary. Avoid unnecessary sharing and adhere to the principle of least privilege. Admin tokens can be deleted from the **Access Tokens** page within your organization settings following the process below.
```

**Review by meagancojocar — APPROVED** [review]
Approving to unblock this addition but I think we should add Admin as a column to the table as well.

File: themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md
Comment (meagancojocar) [inline themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md:112]:
```suggestion
{{% notes type="warning" %}}
Admin organization access tokens have elevated permissions, please use them with caution. 
{{% /notes %}}
```

**Pull request merged**
